src/systems/eu/eu-component/eu-component-global-navigation/eu-component-global-navigation.scss

```diff
@@ -162,6 +165,10 @@
   .ecl-global-navigation__info {
     order: 2;
   }
+
+  .ecl-global-navigation__toggle {
+    padding-bottom: map-get($ecl-spacing, 's');
```

Comment:
I think it should be the opposite: 'm' here and 's' at line 71, don't you think so?

Reply:
well, as we don't have clear specification for this it's quite hard to guess from the screenshots.
Let's put 'm' everywhere and wait for their feedback :)